client/cli/tx.go

```diff
@@ -54,3 +68,114 @@ func NewCmdExecute() *cobra.Command {
 
 	return cmd
 }
+
+// NewCmdSubmitUpgrade schedule an on-chain software upgrade.
+// This command has been adapted from the SDK's `NewCmdSubmitUpgradeProposal`.
+// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/tx.go#L47
```

Comment:
Obviously this is a nit, but do have a small cleanup for this.

```suggestion
// NewCmdSoftwareUpgrade is a helper for scheduling a software upgrade.
//
// This command has been adapted from the Cosmos SDK implementation.
// https://github.com/cosmos/cosmos-sdk/blob/x/upgrade/v0.1.4/x/upgrade/client/cli/tx.go#L46-L133
```